### PR TITLE
Enable Persist BatchNorm in CTL ResNet50 

### PR DESCRIPTION
**Pull request by houtoms**
As in the Keras compile/fit mode (https://github.com/tensorflow/models/blob/master/official/vision/image_classification/resnet_imagenet_main.py#L62), this PR enables the persistent CUDNN batch norm by default for better performance in CTL ResNet50.

fyi @nluehr

### CODE REVIEW

File: official/vision/image_classification/resnet_ctl_imagenet_main.py [hunk 1]
```diff
@@ -197,6 +197,8 @@ def run(flags_obj):
         'mixed_bfloat16')
     tf.compat.v2.keras.mixed_precision.experimental.set_policy(policy)
 
+  common.set_cudnn_batchnorm_mode()
```
[the remark this answers is not in the document]
Comment (houtoms) [reply official/vision/image_classification/resnet_ctl_imagenet_main.py]: Sure. Done. PTAL.
